webpack.config.js

```diff
@@ -35,8 +35,12 @@ module.exports = {
     fs: 'empty'
   },
   externals: {
-    xhr2: 'XMLHttpRequest',
-    xmlhttprequest: 'XMLHttpRequest'
+    'xhr2': 'XMLHttpRequest',
+    'xmlhttprequest': 'XMLHttpRequest',
+    'node-fetch': 'fetch',
+    'text-encoding': 'TextEncoder',
+    'urlutils': 'URL',
+    'webcrypto': 'crypto'
```

Comment:
I'd love to find a way to not include them; haven't been able to get it to work without.